client/cli/src/commands/run_cmd.rs

```diff
@@ -100,6 +99,11 @@ pub struct RunCmd {
 	#[structopt(long = "unsafe-ws-external")]
 	pub unsafe_ws_external: bool,
 
+	/// Override the the maximum payload of RPC server (both http and ws) request and response, in
+	/// megabytes. Default is 15MiB if unset.
+	#[structopt(long = "rpc-max-payload")]
+	pub rpc_max_payload_override: Option<usize>,
```

Comment:
Why talk about "override"? Why not simply "set"? 
```suggestion
	/// Set the the maximum RPC payload size for both requests and responses (both http and ws), in
	/// megabytes. Default is 15MiB.
	#[structopt(long = "rpc-max-payload")]
	pub rpc_max_payload: Option<usize>,
```

Reply:
Agreed, also makes the naming easier and nicer.